meson.build:147: WARNING: Project targets '>= 0.53.0' but uses feature introduced in '0.62.0': dep 'jni' custom lookup.
--- meson.build.orig	2022-10-21 08:51:10.000000000 -0500
+++ meson.build	2022-10-30 04:11:53.000000000 -0500
@@ -12,7 +12,7 @@
     'b_ndebug=if-release',
     'b_lundef=true',
   ],
-  meson_version: '>= 0.53.0'
+  meson_version: '>= 0.62.0'
 )
 
 rubberband_dynamic_library_version = '2.2.1'
